src/main/java/Entities/Group.java

```diff
@@ -1,15 +1,101 @@
 package Entities;
```

Comment:
It looks like your entity classes aren't up to date, which might be what's causing the merge conflicts? Also, I think it's okay for you to have entities in your commit since your use case interacts with them directly.